ge.c

```diff
@@ -49,38 +59,44 @@ int ge_load(struct ge *ge, uint8_t *program, uint8_t size)
     if (program == NULL && size != 0)
         return -1;
 
-    /* When pressing LOAD button, AINI is set. If AINI is set, the state 80
-     * (initialitiation) goes to state c8, starting the loading of the program
-     * (of max 129 words) from one of the peripherc unit. */
+    if (ge->console.buttons.B_LOAD) {
```

Comment:
I'm not sure if checking the buttons here is the right approach.
Should this function in toto be called when the button is pressed? (or reproduce the side effects of pressing the button)